seeds/tickers.js

```diff
@@ -0,0 +1,65 @@
+// TODO: add test suite
+
+function createTickers() {
+  // returns the same array of 100 unique strings (tickers)
+    // each ticker is 3 - 4 characters in length // TODO: vary the ticker length
+    // each ticker is made up of the letters A - Z
+
+    let seed = 1;
```

Comment:
Why is the seed 1? can you add a context comment please. Or what is the effect of changing this value?